packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -188,18 +183,22 @@ contract GenArt721CoreV3_Engine is
     // packed uint: max of 100, max uint8 = 255
     uint8 private _platformProviderPrimarySalesPercentage = 10;
 
-    /// The render provider payment address for all secondary sales royalty
-    /// revenues
-    address payable public renderProviderSecondarySalesAddress;
-    /// Basis Points of secondary sales royalties allocated to the
-    /// render provider
-    uint256 public renderProviderSecondarySalesBPS = 250;
-    /// The platform provider payment address for all secondary sales royalty
-    /// revenues
-    address payable public platformProviderSecondarySalesAddress;
-    /// Basis Points of secondary sales royalties allocated to the
-    /// platform provider
-    uint256 public platformProviderSecondarySalesBPS = 250;
+    /// The default render provider payment address for all secondary sales royalty
```

Comment:
Just to clarify, the only way these could be different on a per project basis is if we updated these and then didn't call `syncProviderSecondaryForProjectToDefaults` for the project right?

Reply:
exactly 👍

Reply:
cool cool, do you think it makes sense to add a clarifying dev comment somewhere that calls this out? I think I might assume these were more readily updatable per project just based on the usual context of the `default` prefix.

Reply:
good call - Hopefully the comments in the update functions help a bit as well, and I've added some clarifying comments surrounding these state variables in [38a57bc](https://github.com/ArtBlocks/artblocks-contracts/pull/1455/commits/38a57bc073edd092bff0013eb6fa2af402a2845b)